src/components/details-name/details-name.styles.scss

```diff
@@ -1,10 +1,9 @@
-$blue: #144078;
-$margin-top: 20px;
+@import "../../utils/variables";
```

Comment:
scss variables moved to their own `utils/_variables.scss` file

Reply:
Thanks! This was bugging me. Updated.